server/src/storage/localfs.rs

```diff
@@ -212,7 +212,8 @@ impl ObjectStorage for LocalFS {
 
     fn absolute_url(&self, prefix: &RelativePath) -> object_store::path::Path {
         object_store::path::Path::parse(
-            format!("{}", self.root.join(prefix.as_str()).display()).trim_start_matches('/'),
+            format!("{}", self.root.join(prefix.as_str()).display())
+                .trim_start_matches(std::path::MAIN_SEPARATOR),
```

Comment:
Using `std::path::MAIN_SEPARATOR` is a better solution as it handles slashes accordingly.